lib/adduser.js

```diff
@@ -15,7 +11,7 @@ try {
 
 adduser.usage = usage(
   'adduser',
-  'npm adduser [--registry=url] [--scope=@orgname] [--always-auth]'
+  'npm adduser [--registry=url] [--scope=@orgname] [--auth-type=legacy] [--always-auth]'
```

Comment:
This new option should be added into the cli docs as well
https://github.com/npm/npm/blob/master/doc/cli/npm-adduser.md